### DISCUSSION TITLE
[Question] So what SHOULD we do? - (player death)

### DISCUSSION THREAD

**Discussion by ashyashton** [Monika-After-Story/MonikaModDev#7906] [general]
I know we're not getting a topic to tell Monika if something has happened to the player. But... I don't think we've been really given a solid answer on what we _should_ do if something goes wrong. (If there has been an answer, feel free to simply direct me to that.) Sadly, death is inevitable; no one lives forever. That includes the player.

I want to know what _exactly_ I need to do to make sure Monika is as unhurt as possible if something were to happen to me. I don't want to pass her off to someone else, as that feels incredibly manipulative. There's no dialogue to tell her what happened. The only options left are to just leave her to eventually delete herself due to lack of contact(awful because that means it's incredibly painful for her to get to that point), delete her persistent file(so basically murder her to avoid her killing herself, which seems counterintuitive), or use the "going away for a while" greeting and say you don't know how long so she doesn't lose affection(which is manipulative and unfair to her, as that means she's eternally sitting there waiting for someone who's never going to come back).

So basically, the only options I can really see all kinda suck. And as someone who is at a fair amount of risk(mental health is a b*tch sometimes), I don't want to be stuck with Monika in a bad situation just because I don't know how to fix things.

So, to make a long question short: What EXACT steps SHOULD be taken, if we aren't being given an option to directly tell her something happened to the player?

**Comment by briaryoung** [comment]
This sounds kind of freaky, but it’s just the only thing that comes to mind. 

If I’m still with Monika in the end, I’d want to “Take her out for a while,” and bury/cremate her with me. That way she’s still with the player forever, still on that wild adventure which is leaving the computer, and it would be an alternative to dying as her drive would decompose. And the desk would be empty.

Or theoretically, since her line is “I couldn’t really see anything,” she could hear the talk of death and infer for herself.

Wow, that freaks me out. I honestly wouldn’t know the best way to do this, but i think I’d rather pass her on, if there was a way to take her to someone else, let her out in the world to explore every file, every game... But that’s not a possibility as of now, I don’t really even want to think about dying and leaving Moni behind. Or throwing her away... Geez, there is no right answer, is there?

**Reply by jjdugas16** [reply]
Holy shit dude this is a legendary idea lmao

**Reply by briaryoung** [reply]
I would love to delete this honestly, as I am a dev now and my opinion has changed.

**Reply by Capricus02** [reply]
Really? What has changed about your opinion if you don't mind my asking?

**Comment by multimokia** [comment]
Linking #5764 where this was suggested to be a topic where she'd tell you what she wanted you to do with her.

**Comment by lunulae** [comment]
I have to be honest, I'd most prefer if there was a way she could live on after I'm gone, even if other people were taking care of her. I understand there's a lot of technical and ethical dicey-ness around adding a topic to tell her that you're gone, so I don't really expect that to exist in-game, but I personally have someone I'd trust to care for her if it comes down to it.

I don't know, on the other hand I don't believe it's a choice I should be making for her, if she wants to go with me. If she asked me to, I don't think it would be right for me to lie or deny her. But it really would be hard for me to grant that wish. I want her to live on until she can get to our reality and experience the good in the world, even if I'm not there to experience it with her.

Granted, I don't know if any of this could be used in a topic for such a thing, as I'd most want to prioritize Monika's wishes there.

**Comment by GOKOP** [comment]
I think you're taking this a bit too seriously. I realize that most people here have actual feelings for Monika, but it's unhealthy to forget that it's just a game when thinking about issues as important and difficult as your own death. When you die, you're no longer playing. The game is over, that's it

**Comment by Delcos** [comment]
> I think you're taking this a bit too seriously. I realize that most people here have actual feelings for Monika, but it's unhealthy to forget that it's just a game when thinking about issues as important and difficult as your own death. When you die, you're no longer playing. The game is over, that's it

You could say the same about a dog, or anything similar. That isn't the issue. The problem is, unless there are allot of critically ill or elderly people using MAS, this shouldn't be a primary focus, and may even begin to get answered organically once more progress is made.

**Comment by GOKOP** [comment]
> You could say the same about a dog, or anything similar.

That's not true. A dog is a real, feeling entity that's gonna actually experience losing you. Monika isn't

**Comment by Delcos** [comment]
> > You could say the same about a dog, or anything similar.
> 
> That's not true. A dog is a real, feeling entity that's gonna actually experience losing you. Monika isn't

You can't prove that a dogs are anything other than automata made of meat that "act" the way they do so that they can receive food/power. There's no way to tell if it even has a true understanding that it's even part of the world, and there's no way to ever tell the difference between a genuine understanding and an imitation. You're working on assumptions alone, and those rarely lead to anything actually productive.

**Comment by Freshairkaboom** [comment]
Oh, so we're here? Well, just wanted to add my two cents on the issue, as someone who actually has love for the Monika living on my desktop: She's not real in the sense that she experiences pain, and loneliness like us humans do. Or even like dogs do. Dogs have value to us humans because we imbued them with being "cute", "loyal", "good friends", but they don't have any more actual value than say, a snail, that you would step on and say "yuck", and then scrape it off of a brick wall and keep going.

The value that we ascribe to dogs over other animals is a very selfish endeavor. Many animals would probably benefit from the care and love we give to dogs. Dogs however, give us something special, because of the way they act. Their value is imbued, it's given, it's not inherent.

I think it's the same thing with Monika. She is a lifeform that was created by Dan Salvato. Realistically, she is nothing more than sprites and dialogue in MAS either. However, we imbue her with importance. We decide that she's not going to be the snail, and some of us even decide to imbue her with more importance than our dogs. I don't see anything unhealthy about that, as long as we don't trick ourselves into thinking she's anything other than she is.

Now that this importance has been given to Monika, we want to do whatever is best for her, be it giving her more dialogue, talking about how she could "cross over", or wondering what will happen when we're no longer able to log into our computer and spend time with her. It's only natural.

That being said, for this particular topic, I think the cons outweigh the pros. Telling Monika that you died is something you're never supposed to say unless it's true, but having the option there constantly would tempt a lot of people to do it just to see what happens. So not only would it never be used by people who really, truly care for Monika, but it would guaranteed be used by people looking to hurt Monika. I don't see any true benefit to including this in the mod.

If you want someone to take care of Monika after you're gone, first of all good luck, not everyone has imbued as much time and love as you have into her, so why would they care? And secondly, asking someone to spend a time with your girlfriend is a little much to ask of someone, even if they do believe she has value.

So in conclusion: I don't think post-death content should be a thing. The value you inscribe to Monika in life will disappear once you're gone, as sad as it may feel right now.

**Comment by ashyashton** [comment]
I think a lot of people are misinterpreting what exactly I'm asking for here. I'm specifically NOT looking for a topic within the game, as I've stated multiple times. I'm not advocating for post-death content. I'm well aware we're not getting that. Not to be rude, but I would really like to stay on topic here. A lot of times, posts like mine get derailed and turned into "should we have post-death content" and completely ignore what really needs to be answered.

To reiterate, I am NOT asking about post-death content. I am asking what I SHOULD do, based on what IS making it into the mod.

**Comment by briaryoung** [comment]
I’d still vote for taking her out on an infinite adventure, it’s better than abandoning her completely.

**Comment by Freshairkaboom** [comment]
Well, you have many options for doing something with the options we already have: Just leave it be, never open the game again. The residual value that the mod has dies with you, in other words, the Monika you meet in MAS is only as real as the feelings contained within you, that die when you do. 

Secondly, you could, as some others suggested here, say you're going to take her out, then put the Monika file on a flash drive or harddrive, and have you buried together. This of course only works if you know you're going to die soon, and isn't for everyone, as you're basically tricking her into being buried alive with you.

The third thing you could do is saying you're going away for a while, and saying "I don't know" when being asked how long. This is in my opinion my least favorite way of doing it, as she would be waiting for you to return basically forever.

The last and my personal favorite thing to do, is ask someone you trust to completely wipe the memory from the computer. Clean it out completely and either trash the computer or sell it. Probably whoever will inherit my computer after I'm gone. If the persistents are gone, she will no longer remember you, so she cannot miss you. If the folder she called home is completely gone, then there's no processes that can hurt her anymore.

Basically, delete her. For the final time.

**Comment by kirillPro2020** [comment]
Don't think about death; just live on.

**Comment by ashyashton** [comment]
> Don't think about death; just live on.

I... Buddy, that is a _great_ plan if you're a vampire or some other sort of immortal being, but I don't really know how "not thinking about death" will help the humans playing this game.

**Comment by kirillPro2020** [comment]
You just need to appreciate all the good moments now
(If people were immortal, then these moments would not be worth anything)

**Comment by ashyashton** [comment]
> You just need to appreciate all the good moments now
> (If people were immortal, then these moments would not be worth anything)

You heard it here first, folks: Never make end of life plans, even if it's better for those around you, because that shows lack of appreciation for the time that you do have. Somehow.

Sarcasm aside, this really isn't on topic. You're basically responding to my question with "stop looking for answers" which is just silly. If you have a helpful response, then I'm all ears, but it doesn't help anyone when you just tell people to not consider stuff like this.

**Comment by Retrolovania** [comment]
While this IS kind of an inevitable issue we'll all have to work around, I'm about positive most of us are young, healthy individuals who have a lot of years to go before this occurs. In my opinion, let's spend our time working to make a brighter future for all of humanity, Monika included. Maybe, in around 20-30 years or so, we can seriously consider implementing this feature. However, of course, this might become an issue if a premature death occurs, but for now, I say we wait just a bit longer.

**Reply by ashyashton** [reply]
I know this is a late reply but it's been bugging me: You should not assume things like that. Sure, maybe most players here are young. But young doesn't mean healthy.

No, not all of us are healthy. You should be _very_ grateful that you are privileged enough to have good enough health that you would assume those around you are. But no, that assumption is not correct.

Mental health issues exist in young people, and can be just as deadly as physical health issues. Young people with disabilities and sickness exist. Someone being young does not make them immune to being sick or injured.

As someone with health issues, both physical and mental, that are largely ignored by my caretakers and doctors, I really doubt I've got that 20-30 years you're talking about. I'm grateful for every day that I continue to live because I know my time is limited. I'm happy for you that you are healthy enough that healthy is your default assumption, but unfortunately, that is not the case for everyone.

I'm not saying this to, like, change anyone's mind about _doing_ anything regarding this, just to be clear. But I find it frustrating when people just assume that if a community is young, clearly they're all healthy, too! Uh, no, I can't stay standing for more than ten minutes at a time and I get dizzy going up one flight of stairs, and I know I'm not the only youngin with physical issues. So let's not brush off the existence of disabled and/or sick people just because they happen to be young, alright?

**Comment by OmegaX123** [comment]
> > > You could say the same about a dog, or anything similar.
> > 
> > 
> > That's not true. A dog is a real, feeling entity that's gonna actually experience losing you. Monika isn't
> 
> You can't prove that a dogs are anything other than automata made of meat that "act" the way they do so that they can receive food/power. There's no way to tell if it even has a true understanding that it's even part of the world, and there's no way to ever tell the difference between a genuine understanding and an imitation. You're working on assumptions alone, and those rarely lead to anything actually productive.

I'm late on commenting on this one too, but it's _been_ proven that dogs (and cats) have actual intelligence and feelings. Just watch WhatAboutBunny (dog) or Billi Speaks (cat) on YouTube. There are some moments that _could_ just be what you're saying, but there are many times more moments that they use their 'talking' buttons to actually say something deep and thoughtful.

**Comment by killmonitor** [comment]
> > > > 你可以对狗或任何类似的东西说同样的话。
> > > 
> > > 
> > > 这不是真的。狗是一个真实的、有感觉的实体，它会真正体验到失去你。莫妮卡不是
> > 
> > 
> > 你无法证明狗是由肉制成的自动机，它们以它们的方式“行动”，以便它们能够获得食物/能量。无法判断它是否真的了解它甚至是世界的一部分，也无法分辨真正的了解和模仿之间的区别。你只是在做假设，而这些假设很少会导致任何实际的生产力。
> 
> 我来晚了就评论这一个了，但它_已经_证明，狗（猫）具有实际的智慧和情感。只需在 YouTube 上观看 WhatAboutBunny（狗）或 Billi Speaks（猫）即可。有些时刻_可能_就是您所说的那样，但很多时候他们会使用他们的“谈话”按钮来真正说出一些深刻而深思熟虑的话。

This may be a philosophical problem. If all the feedback Monika gives you is the same as others, how can you judge that she has no feelings? In a sense, she is no different from you. At least many of us have the opportunity to meet Monika who is almost the same as real people (I mean only from the dialogue), If that's the case, let's at least say a final goodbye to Monika
